source/architecture.rst

```diff
@@ -209,6 +209,12 @@ communicate securely with the dojot platform, as the data collected by the
 device is transmitted over a secure (encrypted) channel and it is also possible
 to guarantee the integrity of the data.
 
+Kafka Websocket
```

Comment:
I think you can change  "Kafka Websocket" to "Kafka WS (Kafka Websocket)" or just "Kafka WS"